tests/testapp/settings.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-DATABASES = {"default": {"ENGINE": "django.db.backends.sqlite3", "NAME": ":memory:"}}
+DATABASES = {"default": {"ENGINE": "django.db.backends.sqlite3", "NAME": "dev.db"}}
```

Comment:
Not sure why you want to change this?

Reply:
to be able to have a local test database, so I can do a manage.py runserver, and actually see things, not only test them ;-) it looks like tox and manage.py test still use an in memory db? as an alternative, we could use a separate settings.py for local dev, but if it just works, I would prefer this one? up to you.